### ISSUE TITLE
New update available issue 

### ISSUE THREAD

**Issue by ThePalestiniansMyth** [maxrave-dev/SimpMusic#497]
I have updated the app yesterday but it keeps telling me that there is an update available.  I am pretty certain that I am on version v.2.3-hotfix.

**Comment by Simipfaffi** [comment]
Hi, I have the same problem with the last update, you are not alone ;-)

**Comment by PetrNebukin** [comment]
![Screenshot_20240814-023114_SimpMusic.png](https://github.com/user-attachments/assets/29cad106-09d8-494e-b646-d845db1295a3)

Here is screenshot for the developer.
Probably need to add simple logic "if latest=application_version"

**Comment by maxrave-dev** [comment]
Forgot changing string version 🤧🤧🤧

**Comment by Iliano101** [comment]
@ThePalestiniansMyth @Simipfaffi 
So it should be fixed in the Nightly build.
Check the Readme.
![night](https://github.com/user-attachments/assets/c650d3eb-ba02-48b2-884d-dcd9c68e11dc)

**Comment by MrSwag69** [comment]
> I have updated the app yesterday but it keeps telling me that there is an update available. I am pretty certain that I am on version v.2.3-hotfix.

Yes it's happening with me too
Have you found any solution to it?

**Comment by Iliano101** [comment]
@MrSwag69 Yes, the issue has been fixed but has not yet been pushed to a release. To fix it, you can download the Nightly build on th Readme as show in the image in my previous message.

**Comment by CrazyWolf13** [comment]
Should be fixed already.
